File: 백준/Bronze/10871. X보다 작은 수/X보다 작은 수.cc
#include <stdio.h>
int main(){
    int N, X, SOO;
    scanf("%d %d", &N, &X);
    for(int i = 0; i < N; i ++){
        scanf(" %d", &SOO);
        if(SOO < X){
            printf("%d ", SOO);
        }
    }
}

File: 백준/Bronze/10871. X보다 작은 수/X보다 작은 수.java
import java.util.Scanner;

public class Main{
    public static void main(String args[]){
        Scanner sc = new Scanner(System.in);
        int N, X, SOO;
        N = sc.nextInt();
        X = sc.nextInt();
        for (int i= 0; i < N; i++){
            SOO = sc.nextInt();
            if(SOO < X){
                System.out.printf("%d ", SOO);
            }
        }
    }
}